utility/src/main/java/org/oppia/android/util/networking/BUILD.bazel

```diff
@@ -5,12 +5,34 @@ Package for networking utilities
 load("@dagger//:workspace_defs.bzl", "dagger_rules")
 load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
 
+kt_android_library(
+    name = "connection_status",
+    srcs = [
+        "ConnectionStatus.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+)
+
 kt_android_library(
     name = "network_connection_util",
     srcs = [
         "NetworkConnectionUtil.kt",
     ],
     visibility = ["//:oppia_api_visibility"],
+    deps = [
+        ":connection_status",
+    ],
+)
+
+kt_android_library(
+    name = "network_connection_debug_util",
+    srcs = [
+        "NetworkConnectionDebugUtil.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
```

Comment:
Hmm. I'm wondering if we could introduce a new oppia_debug_visibility that's basically oppia testing visibility + specific subpackages that are allowed to access debug utilities. WDYT? That will help isolate these debug utilities to exactly where they should be used.

Reply:
I think this makes sense to have separate debug visibility. I have introduced a new `oppia_debug_visibility`. PTAL if it is correct or not. Thanks.